src/main/java/gift/product/util/CertifyUtil.java

```diff
@@ -0,0 +1,88 @@
+package gift.product.util;
+
+import gift.product.model.Member;
+import gift.product.service.MemberService;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.UnsupportedJwtException;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import java.security.Key;
+import java.util.Base64;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CertifyUtil {
+
+    private final String secretKey = "Yn2kjibddFAWtnPJ2AFlL8WXmohJMCvigQggaEypa5E=";
+    private final Key key = Keys.hmacShaKeyFor(secretKey.getBytes());
+    private final MemberService memberService;
+
+    public CertifyUtil(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    public Member encryption(String email, String password) {
+        return new Member(Base64.getEncoder().encodeToString(email.getBytes()),
+            Base64.getEncoder().encodeToString(password.getBytes()), 0);
+    }
```

Comment:
```suggestion
     public Member encryptMember(String email, String password) {
        return new Member(
                Base64.getEncoder().encodeToString(email.getBytes()),
                Base64.getEncoder().encodeToString(password.getBytes()),
                0
        );
    }
```
코드 스타일과 메서드 명을 수정하는게 어떨까요?